sdk/tools/cabman/cabinet.h

```diff
@@ -29,25 +30,28 @@
 #endif
 
 #if defined(_WIN32)
-#define DIR_SEPARATOR_CHAR '\\'
-#define DIR_SEPARATOR_STRING "\\"
+    #define DIR_SEPARATOR_CHAR '\\'
+    #define DIR_SEPARATOR_STRING "\\"
```

Comment:
Why is this indented?

Reply:
Indentation removed in commit 299cedeb13

Reply:
For what it's worth, we seem to have three indentation styles for preprocessor directives in use throughout the codebase:
1. No indentation at all
2. Indenting like @dfbag7 did
3. Indenting the directive but not the hash (e.g. "#    define")

I personally prefer the style @dfbag7 used. As our Coding Style guidelines give no rule regarding that, I use it myself in some modules.
However, it's up to you guys what shall be used for cabman in the future.